lib/restaurant.rb

```diff
@@ -0,0 +1,46 @@
+class Restaurant
+
+attr_accessor :basket
+
+  def initialize
+    puts "To see the menu use the method: 'see_menu'"
+    @basket = []
+  end
+
+  def menu
+    @menu = {1 => ['Margarita:', 5.00], 2 => ['Meat_Feast:', 6.00], 3 => ['Hawaiin:', 5.50],
+            4 => ['BBQ_Chicken:', 6.00], 5 => ['Vegetarian:', 5.50], 6 => ['Full_Monty:', 8.00],
+            7 => ['Ham_and_Cheese:', 4.00], 8 => ['Pepperoni:', 5.50], 9 => ['Chips:', 3.00]}
+  end
+
+  def see_menu
```

Comment:
Assignment Branch Condition size for see_menu is too high. [64/15]<br>Inconsistent indentation detected.